Makefile

```diff
@@ -44,7 +44,7 @@ all: $(TARGET)
 
 run: $(TARGET)
 ifeq ($(ARCH), lin)
-	LD_LIBRARY_PATH=dep/lib ./$<
+	LD_LIBRARY_PATH=dep/lib:dep/lib64 ./$<
```

Comment:
If libraries are being put in lib64, we need to fix that instead. Everything should go in dep/lib

Reply:
ok, let me see if I can get that done